lib/phoenix_html/form.ex

```diff
@@ -531,10 +533,14 @@ defmodule Phoenix.HTML.Form do
       {prompt, opts} -> {content_tag(:option, prompt, value: ""), opts}
     end
 
-    opts =
-      opts
-      |> Keyword.put_new(:id, id_from(form, field))
-      |> Keyword.put_new(:name, name_from(form, field))
+    opts = Keyword.put_new(opts, :id, id_from(form, field))
+      
+    opts = case Keyword.pop(opts, :multiple) do
```

Comment:
Please indent this as (note the lack of spaces after `{` and `}`):

    opts =
      case Keyword.get(opts, :multiple, false) do
        false -> ...
        true -> ...
      end
      
Notice you don't need to pop the value because Phoenix will do the proper thing if it is true (keep it) or false (remove it).